### PR TITLE
Open workspace should only be in create project and the generic ui

### PR DESCRIPTION
**Pull request by NetaMolcho**

### CODE REVIEW

**Review by slavik-lvovsky — CHANGES_REQUESTED** [review]
Your code looks very good.
The problem is that sub-generator can also be of project type and if the wizard runs only such sub generator your logic will not work.
Let's discuss it.

File: backend/src/youi-adapter.ts [hunk 1]
```diff
@@ -49,7 +49,7 @@ export class YouiAdapter extends TerminalAdapter {
         try {
           return await cb(result); // eslint-disable-line @typescript-eslint/await-thenable
         } catch (err) {
-          this.youiEvents.doGeneratorDone(false, (_.get(err, "message", 'Yeoman UI detected an error')));
+          this.youiEvents.doGeneratorDone(false, (_.get(err, "message", 'Yeoman UI detected an error')), false);
```
Comment (slavik-lvovsky) [inline backend/src/youi-adapter.ts:52]: Chane it to Application Wizard

**Review by slavik-lvovsky — CHANGES_REQUESTED** [review]

**Comment by slavik-lvovsky** [comment]
I have added a new command that enables running any main/sub generator

**Review by slavik-lvovsky — APPROVED** [review]

File: backend/package.json [hunk 2]
```diff
@@ -58,6 +58,10 @@
 			{
 				"command": "exploreGenerators",
 				"title": "Explore and Install Generators"
+			},
+			{
+				"command": "runAnyGenerator",
```
Comment (tomer-epstein) [inline backend/package.json:63]: runGenerator sounds better

**Review by slavik-lvovsky — CHANGES_REQUESTED** [review]
generator-license does not work in vscode
It fails with exception.
We need to understand why. It works in BAS
It does not relates to your PR, it is an old bug and we have to fix it.

**Review by slavik-lvovsky — APPROVED** [review]

**Pull request merged**